_unittest/test_20_HFSS.py

```diff
@@ -568,7 +568,7 @@ def test_24_create_curvilinear(self):
     def test_25_create_parametrics(self):
         self.aedtapp["w1"] = "10mm"
         self.aedtapp["w2"] = "2mm"
-        setup1 = self.aedtapp.parametrics.add({"w1": "LIN 0.1mm 20mm 0.2mm"})
+        setup1 = self.aedtapp.parametrics.add("w1", 0.1, 20, 0.2, "LinearStep")
         assert setup1
         assert setup1.add_variation("w2", "0.1mm", 10, 11)
```

Comment:
I understand the logic, but what I find confusing is having 3 methods to do the same thing. parametrics.add,  parametrics.set_variation, optimization.add_variable.